c++/src/capnp/layout.c++

```diff
@@ -2951,7 +2951,7 @@ ListReader ListReader::imbue(CapTableReader* capTable) const {
   return result;
 }
 
-bool ListReader::isCanonical(const word **readHead) {
+  bool ListReader::isCanonical(const word **readHead, const WirePointer *ref) {
```

Comment:
Oops. Fixed.

I think emacs wanted the indentation because of the enclosing `namespace {`.